backend/corpora/common/corpora_orm.py

```diff
@@ -230,6 +270,10 @@ class DbDataset(Base):
     created_at = Column(DateTime, nullable=False, server_default=DEFAULT_DATETIME)
     updated_at = Column(DateTime, nullable=False, server_default=DEFAULT_DATETIME)
 
+    artifacts = relationship("DbDatasetArtifact", back_populates="dataset")
```

Comment:
We probably also want to add the `cascade` argument to inform that children in a relationship should be affected when the parent changes (i.e. gets deleted). Ditto in all the child relationships in this file.

Reply:
Good idea, I'll add to all of the children in this class.

Reply:
On second thought I think this should wait until we are handling deletion, since this feature will not be tested until then.